### PR TITLE
[TinyMCE] Add `paste_preprocess` and `pagebreak_separator` plugins

### PR DESCRIPTION
**Pull request by wafuwafu13**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/tinymce/index.d.ts [hunk 1]
```diff
@@ -107,6 +108,32 @@ export interface Settings {
 
     paste_data_images?: boolean;
 
+    paste_as_text?: boolean;
```
Comment (wafuwafu13) [inline types/tinymce/index.d.ts:111]: [docs] https://www.tiny.cloud/docs-4x/plugins/paste/#paste_preprocess

File: types/tinymce/index.d.ts [hunk 2]
```diff
@@ -107,6 +108,32 @@ export interface Settings {
...
+
+    paste_retain_style_properties?: string;
+
+    paste_merge_formats?: boolean;
+
+    paste_convert_word_fake_lists?: boolean;
+
+    paste_remove_styles_if_webkit?: boolean;
+
+    pagebreak_separator?: string;
```
Comment (wafuwafu13) [inline types/tinymce/index.d.ts:133]: [docs] https://www.tiny.cloud/docs-4x/plugins/pagebreak/#pagebreak_separator

File: types/tinymce/index.d.ts [hunk 3]
```diff
@@ -1086,7 +1113,7 @@ export namespace dom {
 
         getNode(): Element;
 
-        getRng(w3c: boolean): Range;
+        getRng(): Range;
```
Comment (wafuwafu13) [inline types/tinymce/index.d.ts:1116]: [docs] https://www.tiny.cloud/docs-4x/api/tinymce.dom/tinymce.dom.selection/#getrng

**Review by Parent5446 — APPROVED** [review]

**Comment by wafuwafu13** [comment]
Ready to merge

**Pull request merged**